Automattic/jetpack
# Build: First pass at minifying module JS

As part of a front-end performance audit that @gravityrail and I are doing, this PR will start minifying all front-end assets that are loaded from the shortcodes module.

To test:

- Checkout branch
- `yarn build`
- Ensure that license is at top of the output for this command:
    - `more _inc/build/shortcodes/js/jmpress.min.js`
- Test the following shortcodes:
    - brightcove
    - gist
    - instagram
    - presentation
    - quiz
    - recipe
    - slideshow
- Ensure there are no JS errors on the frontend and that shortcodes are still functional
- In `wp-config.php`, add `define( 'SCRIPT_DEBUG`, true );` and check that non-minified files are now enqueued by checking the source

Note: I am removing `modules/shortcodes/js/jmpress.min.js` from source control since we'll now build it.
